src/styles/blocks/watch.scss

```
@@ -0,0 +1,54 @@
+.stop-watch {
+  @include square($size-watch);
+
+  &__arrow {
+    position: relative;
+    top: 50%;
+    left: 50%;
+    transform-origin: left center;
+
+    &--seconds {
+      height: $height-seconds-arrow;
+      width: $width-seconds-arrow;
+      border-end-end-radius: 30%;
+      background-color: $color-seconds-arrow;
+      animation: watch-animation 
+        60s 
+        linear 
+        infinite;
+
+      &::after {
+        content: "";
+        position: relative;
+        bottom: 13px;
+        right: 7px;
+        display: block;
+        background-color: $color-start-point;
+        @include circle($start-point-size);
+      }
+    }
+
+    &--minutes {
+      position: relative;
+      z-index: 0;
+      height: $height-minute-arrow;
+      width: $width-minute-arrow;
+      border-radius: 45px;
+      background-color: $color-minute-arrow;
+      animation: watch-animation 
+        3600s
+        linear 
+        infinite;
+    }
+  }
+
+  @keyframes watch-animation {
+    0% {
+      transform: rotate(270deg);
+    }
+
+    100% {
+      transform: rotate(630deg);
+    }
```

```suggestion
  @keyframes watch-animation {
    0% {
      transform: rotate(0);
    }

    100% {
      transform: rotate(360deg);
    }
```
Try to make it work. `transfrom-origin: bottom` may help you